kips/kip-1.md

```diff
@@ -186,33 +186,33 @@ contract UserPot is RBACWithAdmin, UserPotInterface, ERC165, UpgradeableInterfac
     require(bal >= _deposit, 'you need to pay more to register for event');
     _updateUserData(_user, msg.sender, bal - _deposit);      
   }
+
+  function withdraw() external {
+    uint256 bal = calculatePayout(msg.sender);
+    msg.sender.transfer(bal);
+    _updateUserData(msg.sender, address(0), 0);
+  }
   
-    function withdraw() external {
-        uint256 bal = calculatePayout(msg.sender);
-        msg.sender.transfer(bal);
-        _updateUserData(msg.sender, address(0), 0);
+  function _updateUserData(address _user, address _newEvent, uint256 _newBalance) internal {
```

Comment:
I don't think I understand this logic. Can you write down what this function does as a comment?

Reply:
Added comments to this function.